docker/distelli-manifest-nightly.yml

```diff
@@ -0,0 +1,9 @@
+pe-and-platform/puppetserver-nightly:
```

Comment:
This file matches docker/distelli-manifest.yml except for this line, and unfortunately you can't have two pipelines share this file due to this line needing to match the pipeline name and the pipeline name needing to be unique